NoiseStudies/Macros/remap.txt

```diff
@@ -0,0 +1,35 @@
+channels[0][0][0][1] = channels_before_reorg[0][0][11][1];
+	channels[0][0][1][1] = channels_before_reorg[0][0][15][1];
```

Comment:
is this used for other code or is it just for debugging/documentation

Reply:
This was code Josh gave me but I never used it. It should be deleted now